### DISCUSSION TITLE
Announcing `bevy-persistent` v0.3.0

### DISCUSSION THREAD

**Discussion by umut-sahin** [bevyengine/bevy#9270] [Show and tell]
Hello Bevy Community!

I'm happy to announce v0.3.0 of `bevy-persistent`, a Bevy helper to easily manage resources that need to persist across game sessions.

- Crate: https://crates.io/crates/bevy-persistent
- Documentation: https://docs.rs/bevy-persistent
- Repository: https://github.com/umut-sahin/bevy-persistent
- Examples: https://github.com/umut-sahin/bevy-persistent/tree/main/examples

This release unlocks the library for more use cases and improves reliability:

- Bevy v0.11
- Error handling (see [PersistenceError](https://docs.rs/bevy-persistent/latest/bevy_persistent/error/enum.PersistenceError.html))
- Reverting (you can learn more about it [here](https://github.com/umut-sahin/bevy-persistent#reverting))
- Unloading/Reloading (you can learn more about it [here](https://github.com/umut-sahin/bevy-persistent#unloadingreloading))
- and more! (see [release notes](https://github.com/umut-sahin/bevy-persistent/releases/tag/v0.3.0))

Always feel free to suggest more ideas, create bug reports, give feedback, and contribute!

**Comment by alice-i-cecile** [comment]
You should submit a PR to get it listed on Bevy Assets! https://github.com/bevyengine/bevy-assets

**Reply by umut-sahin** [reply]
It's added a few weeks ago.

https://github.com/bevyengine/bevy-assets/blob/main/Assets/Persistence/bevy-persistent.toml

Thanks for the suggestion!
